src/components/virtualSessions/NewVirtualSession.tsx

```diff
@@ -0,0 +1,208 @@
+import * as React from "react";
+import styled from "styled-components";
+import blockOne from "../../assets/images/virtualsessions/1.png";
+import blockTwo from "../../assets/images/virtualsessions/2.png";
+import blockThree from "../../assets/images/virtualsessions/3.png";
+import { VirtualSessionInterface } from "../../models/VirtualSession";
+import { ServiceResolver } from "../../services/ServiceResolver";
+import { SessionStorageHelper } from "../../helpers/SessionStorageHelper";
+import { JwtTokenHelper } from "../../helpers/JwtTokenHelper";
+import { AuthContext } from "../../contexts/AuthContext";
+
+
+const VirtualSessionsWrapper = styled.div`
+  font-family: "Lato", sans-serif;
+  color: #212529;
+  padding-bottom: 20px;
+
+  li {
+    display: flex;
+    align-items: flex-start;
+    
+
+    img { 
+      margin-right: 1rem;
+    }
+  }
+`
+const PreferenceTitle = styled.div`
+  font-size: 16px;
+  font-weight: bold;
+`
+
+const SubmitButton = styled.button`
+  font-family: "Lato", sans-serif;
+
+  background-color: #FCC600;
+  color: #000000;
+  border-radius: 4px;
+  padding: 11px 56px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 14px;
+  border: none;
+
+  :disabled {
+    border: 1px solid #999999;
+    background-color: #cccccc;
+    color: #666666;
+  }
+`;
+
+interface NewVirtualSessionState {
+  timePreferenceOne?: string;
+  timePreferenceTwo?: string;
+  timePreferenceThree?: string;
+  submitDisabled?: boolean;
+}
+
+export class NewVirtualSession extends React.Component<{}, NewVirtualSessionState> {
+  public apiService = new ServiceResolver().ApiService();
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      timePreferenceOne: undefined,
+      timePreferenceTwo: undefined,
+      timePreferenceThree: undefined,
+      submitDisabled: true
+    }
+  }
+
+  handleDateTimePreferenceChange = (e: any) => {
+    this.setState({
+      [e.target.name]: new Date(e.target.value).toISOString()
+    }, this.handleDisableChange)
+  }
+
+  handleDisableChange = () => {
+    const { timePreferenceOne, timePreferenceTwo, timePreferenceThree } = this.state;
+    const submitDisabled = timePreferenceOne === undefined || timePreferenceTwo === undefined || timePreferenceThree === undefined;
```

Comment:
Yeah good thought here. plus the default time/date input type doesn't seem to be super user friendly so it would avoid having a person require to input all data. i'll explore this and see how it looks w/ these changes.